src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -398,8 +398,8 @@ public function testFormatWithDateTimeZone()
             $this->markTestSkipped('Only in PHP 5.5+ IntlDateFormatter allows to use DateTimeZone objects.');
         }
 
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
+        if (defined('HHVM_VERSION_ID') || PHP_VERSION_ID <= 50509) {
```

Comment:
Well, currently, we have 2 different skipping messages: 1 for PHP saying it needs PHP 5.5.10+ (assuming it gets fixes), and the other one for HHVM linking to the HHVM issue.
It makes more sense to use keep the PHP check in a single place rather than 2